packages/overlays/test/LocalOverlayController.test.js

```diff
@@ -122,6 +122,54 @@ describe('LocalOverlayController', () => {
     });
   });
 
+  describe('nodes', () => {
+    it('accepts HTML Elements (contentNode) to render content', async () => {
+      const invokerNode = await fixture(html`
+        <div role="button" id="invoker">Invoker</div>
+      `);
+
+      const node = document.createElement('div');
+      node.innerHTML = '<div id="content">Content</div>';
+
+      const ctrl = new LocalOverlayController({
+        contentNode: node,
+        invokerNode,
+      });
+      const el = await fixture(html`
+        <div>
+          ${ctrl.invoker} ${ctrl.content}
```

Comment:
`ctrl.invoker` is deprecated? should be `invokerNode`?

Reply:
Aahhhh I didn't catch that memo, I kinda copied the template from some other test :P